src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -77,7 +81,7 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseRep
      * @return the ResponseEntity with status 201 (Created) and with body the new complaints
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
-    @PostMapping("/complaints")
+    @PostMapping("complaints") // TODO: should be participations/{participationId}/results/{resultId}/complaints
```

Comment:
The path in the comment needs to be adjusted as well I suppose, but I guess we can leave this to after the TODO is resolved.